Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -295,6 +295,27 @@ Advanced Configuration
 This section covers a few advanced configuration modes for the kube-proxy replacement
 that go beyond the above Quick-Start guide and are entirely optional.
 
+Policy Enforcement
+******************
+
+When accessing a service from outside the cluster, the :ref:`arch_id_security`
+assignment depends on the routing and the kube-proxy replacement configuration.
+
+In the tunneling mode (i.e., ``--tunnel=vxlan`` or ``--tunnel=geneve``), the request
+to the service will have the ``reserved:world`` security identity. This allows a user to
+use a network policy with the ``fromEntities: cluster`` label selector.
```

Comment:
The second sentence here doesn't make sense. `fromEntities: world` would select the `reserved:world` identity. `fromEntities: cluster` explicitly *does not* match `reserved:world` identity.

Reply:
What I wanted to say with that sentence is that users can expect a consistent behavior when using network policy with `fromEntities=cluster` selector. But after re-reading I can see how it can make no sense :-)